src/main/java/com/atlassian/db/replica/api/DualConnection.java

```diff
@@ -44,7 +44,7 @@
  * Falls back to the main database if it cannot use a replica.
  */
 public final class DualConnection implements Connection {
-    public static final String CONNECTION_CLOSED_MESSAGE = "This connection has been closed.";
+    private static final String CONNECTION_CLOSED_MESSAGE = "This connection has been closed.";
```

Comment:
Ah yes, [my late feedback](https://github.com/atlassian-labs/db-replica/pull/6#discussion_r560155305) 😄
Thanks for the follow-through!